karaf/apache-brooklyn/src/main/resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -18,29 +18,74 @@
 ################################################################################
```

Comment:
@geomacy Not sure why this comment is in the brooklyn repo - I thought I made the comment in a private AMP repo on a different PR?! Deleting these comments as they have nothing to do with this brooklyn code/PR.